handbook/engineering/distribution/goals.md

```diff
@@ -67,6 +67,16 @@ Creating a new release for our deployments is currently a semi-automated process
   - Document project and folder usage guidelines.
   - Set spending limits for dynamic environments.
 
+### Improve Sourcegraph monitoring developer experience
```

Comment:
This should be more tightly scoped. There isn't a clear end here. We could always improve the monitoring experience.

Reply:
This is the "monitoring development" experience, not the general monitoring experience. I see your point here, but I believe this should defined by the desired outcomes more than its title.
Still it could be possible you dont think the outcomes tighten that scope enough.

Reply:
I agree that both the title and the documented outcomes are not clear and scoped. A reader needs to be able to understand what done looks like. A problem statement is also missing.